src/simulation/dynamics/_GeneralModuleFiles/THROperation.h

```diff
@@ -33,8 +33,9 @@ THROperation
 //@endcond
 {
     uint64_t fireCounter;                //!< (-) Number of times thruster fired
-    double ThrustFactor;                 //!< -- Current Thrust Percentage
+    double ThrustFactor;                 //!< -- Current thrust percentage due to ramp up/down
     double IspFactor;                    //!< -- Current fractional ISP
+    double BlowDownFactor = 1.0;               //!< -- Current thrust percentage due to tank blow down
```

Comment:
:small_red_triangle_down: `blowDownFactor`

Reply:
This has been changed to a list of thruster blow down polynomial coefficients